programs/castle-vault/src/adapters/port.rs

```diff
@@ -136,7 +260,14 @@ impl<'info> LendingMarket for PortAccounts<'info> {
     }
 
     fn lp_tokens_in_vault(&self) -> u64 {
-        self.vault_port_lp_token.amount
+        // TODO can this be improved?
```

Comment:
I think you can make the type of `vault_port_stake_account` a `Box<Account<'info, PortStakeAccount>>` in the `PortAccounts` struct and then just access the `deposited_amount` easily.